eth/ethtest/mock_client.go

```diff
@@ -132,6 +136,14 @@ func (m *MockClient) EstimateGas(
 	return args.Get(0).(uint64), args.Error(1)
 }
 
+func (m *MockClient) GetExpiry(
+	ctx context.Context,
+	addr common.Address,
+) (uint32, error) {
+	args := m.Called(ctx, addr)
+	return args.Get(0).(uint32), args.Error(1)
```

Comment:
I think we should be using uint64 or int64 for the timestamps https://golang.org/pkg/time/#Time.Unix